### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Xiang Dai <764524258@qq.com>

### CODE REVIEW

**Review by nik9000 — CHANGES_REQUESTED** [review]
This is quite long to review. I think it'd be nice if you could break it up a bit. Maybe also break out the removal of trailing spaces as well.

**Comment by daixiang0** [comment]
@nik9000 I want to split them then maybe too many pr, so i put them together.
I have a tool to clean up white up safely, i would revert this commit first.

File: docs/plugins/mapper-annotated-text.asciidoc [hunk 1]
```diff
@@ -119,7 +119,7 @@ PUT my_index/_doc/1
 
 PUT my_index/_doc/2
 {
-  "my_field": "[Jeff Beck](Jeff+Beck&Guitarist) plays a strat"<2>
+  "my_field": "[Jeff Beck](Jeff+Beck&Guitarist) plays a start"<2>
```
Comment (nik9000) [inline docs/plugins/mapper-annotated-text.asciidoc]: This one refers to [fender stratocaster](https://en.wikipedia.org/wiki/Fender_Stratocaster).

File: docs/reference/mapping/types/percolator.asciidoc [hunk 2]
```diff
@@ -359,7 +359,7 @@ GET /test_index/_search
     "percolate" : {
       "field" : "query",
       "document" : {
-        "body" : "Bycicles are missing"
+        "body" : "Bicycles are missing"
```
Comment (nik9000) [inline docs/reference/mapping/types/percolator.asciidoc]: I think this one might be intentional

**Review by nik9000 — CHANGES_REQUESTED** [review]
I left two comments about corrections that you made that I think weren't needed. Otherwise this looks good. Could you revert those two change?

**Comment by nik9000** [comment]
@elasticmachine, ok to test

**Review by nik9000 — APPROVED** [review]
Seems right to me. Let's see what CI thinks:

@elasticmachine, ok to test.

**Comment by nik9000** [comment]
CI seemed to like it. I've merged it. It should show up on the site in about an hour for the master branch. I'll open PRs to backport it to 7.x and 7.5 so it shows up there too.

**Pull request merged**